seq2seq/dataset/fields.py

```diff
@@ -10,10 +10,10 @@ def __init__(self, **kwargs):
 
         if kwargs.get('batch_first') is False:
             logger.warning("Option batch_first has to be set to use pytorch-seq2seq.  Changed to True.")
-        kwargs['batch_first'] = True
-        if kwargs.get('batch_first') is False:
+            kwargs['batch_first'] = True
+        if kwargs.get('include_lengths') is False:
             logger.warning("Option include_lengths has to be set to use pytorch-seq2seq.  Changed to True.")
-        kwargs['include_lengths'] = True
```

Comment:
`kwargs['include_lengths']` has to be set and it has to be `True`.  Your change doesn't make sure that it has a value, which leads to the test failure.

Reply:
oh, you are right